### ISSUE TITLE
commons-fileupload is pinned to vulnerable version

### ISSUE THREAD

**Issue by whilp** [awslabs/aws-serverless-java-container#123]
* Framework version: 0.9.1
* Implementations: Jersey / Spring / Spring Boot / Spark

## Scenario

We run a regular analysis of our dependencies and noticed that the version of `commons-fileupload` required by `serverless-java-container` has a CVE open:

https://nvd.nist.gov/vuln/detail/CVE-2016-1000031

This CVE is under reanalysis, but the issue was acknowledged and fixed upstream.

## Expected behavior

serverless-java-container does not require versions of dependencies with known vulnerabilities.

## Actual behavior

commons-upload is pinned to 1.3.2:

https://github.com/awslabs/aws-serverless-java-container/blob/211224eb7296eb083552ea722801f144c8e15220/aws-serverless-java-container-core/pom.xml#L52-L57

**Comment by SAPessi** [comment]
Thanks @whilp, I have updated to the [latest version for the 1.0 release](https://github.com/awslabs/aws-serverless-java-container/blob/core/aws-serverless-java-container-core/pom.xml#L56) and also [added the dependency check](https://github.com/awslabs/aws-serverless-java-container/blob/core/aws-serverless-java-container-core/pom.xml#L111) maven plugin at build time for all modules.

Closing this for now. Given the 2.0 severity rating, I don't think it's worth publishing a patch release of serverless java container - 1.0 is due for release in a couple of weeks. Let me know if you disagree.
